# HG changeset patch
# Parent a63230391776a0dbb642f8fc2f7467c638559d3b
# User Ammler
# Date 1302623864 -7200

Don't add a "M" to version string so we can join MP servers

diff --git a/findversion.sh b/findversion.sh
--- a/findversion.sh
+++ b/findversion.sh
@@ -141,10 +141,6 @@
 	REV_NR=""
 fi
 
-if [ "$MODIFIED" -eq "2" ]; then
-	REV="${REV}M"
-fi
-
 CLEAN_REV=${REV}
 
 if [ -n "$BRANCH" ]; then
